### ISSUE TITLE
Fix position pagination area when scrolling

### ISSUE THREAD

**Issue by anselmbradford** [codeforamerica/ohana-web-search#169]
The pagination / search results summary header should be `position:fixed` when scrolling beyond it's location to enable the user to see the current pagination count and search summary at all times. This requires a JavaScript solution.
